### ISSUE TITLE
Date Picker year selectors don't report as buttons to accessibility.

### ISSUE THREAD

**Issue by darrenaustin** [flutter/flutter#96545]
When selecting a year in the Date Picker, the entries in the year picker don't announce themselves as 'buttons' to the accessibility frameworks. On Android/Talkback they are announced as '2021 double tap to select', but on iOS/VoiceOver it just announces '2021'. To make this easier to use, it should announce the item as a button.

Internal google issue: b/213421823

**Comment by darrenaustin** [comment]
Fixed with #96546.
